### PR TITLE
Lit6585 rh pentecost

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
https://github.com/BetaMasaheft/Documentation/issues/2018

### CODE REVIEW

File: new/LIT6583RHMaryam.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
```
Comment (thea-m) [inline new/LIT6583RHMaryam.xml:1]: Did some branch confusion happen here? LIT6583RHMaryam was created through https://github.com/BetaMasaheft/Works/pull/764/files as well. If this file here is a redundant doublet, I think the best would be if you deleted it from this branch.

**Review by thea-m — CHANGES_REQUESTED** [review]
Logistical question :) The other record is fine. Just notice the quick correction I carried out with the wrong nesting of the bibliography and edition. This happens in some records, as Zhenia pointed out (we should have noticed before, our mistake)

File: new/LIT6583RHMaryam.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply new/LIT6583RHMaryam.xml:1]: Sorry, as usual I don't know how to sort these branch problems... Do you mean that I should switch to branch LIT6585RHPentecost and then delete the file LIT6583RHMaryam from my GitHub folder?

Comment (thea-m) [reply new/LIT6583RHMaryam.xml]: Yes, exactly, that was the problem. I did it now quickly, sorry this got lost among other things for so long.

**Review by thea-m — APPROVED** [review]

**Pull request merged**